appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -728,6 +727,12 @@ private static int upgradeClockProperties(Map<String, JSONValue> componentProper
       // No properties need to be added to upgrade to version 3.
       srcCompVersion = 3;
     }
+    if (srcCompVersion < 4) {
+      // The behaviour of the previous versions are not FixedInterval
+      componentProperties.put("FixedInterval", new ClientJsonString("False"));
```

Comment:
Delete this line. Because the default value is False, it isn't explicitly added to a project. So the upgrader is effectively a noop here.

Reply:
Nahh, the default value of new components is true

Reply:
I stand corrected.

Reply:
As I try to import a old version project (from mit server), the FixedInterval is automatically set to true (checked) with out this line. Did I do any thing wrong?

I just:
* Commented this line:
  ![image](https://user-images.githubusercontent.com/22613139/37207852-d3e55a50-23d9-11e8-8a63-2423fd2da9b4.png)
* upgrade the YoungAndroidVersion locally
  ![image](https://user-images.githubusercontent.com/22613139/37207844-c9a1f17a-23d9-11e8-8dbb-1779baebd2fa.png)

And i got this after imported:
![image](https://user-images.githubusercontent.com/22613139/37207869-e931f184-23d9-11e8-9a23-93653a0af617.png)

It does not work as expected...

With this line i got this which is correct:
![image](https://user-images.githubusercontent.com/22613139/37208016-b7ee880c-23da-11e8-94e8-6a6e7b109f46.png)